deel-3-webpagina/index.html

```diff
@@ -7,10 +7,25 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
+    <nav><h2 class="analytica">ANALYTICA</h2>
+        <div class="links">
+            <a class="about-us" href="#">About us</a>
+            <a class="strategy" href="#">Strategy</a>
+            <a class="prices" href="#">Prices</a>
+            <a class="contact" href="#">Contact</a>
+        </div>
+    </nav>
+    <main>
+        <div class="titles">
         <h1>All Your Business Data in One Place</h1>
         <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
             analytics tools.</h3>
-    </div>
+        </div>
+        <div class="red-boxes">
+        <p class="box-1">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus ad at beatae dolorum, ea eligendi explicabo in iure nesciunt nihil odio odit qui reiciendis sapiente tempore velit veritatis vero voluptatum.</p>
+        <p class="box-2">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus ad at beatae dolorum, ea eligendi explicabo in iure nesciunt nihil odio odit qui reiciendis sapiente tempore velit veritatis vero voluptatum.</p>
+        <p class="box-3">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus ad at beatae dolorum, ea eligendi explicabo in iure nesciunt nihil odio odit qui reiciendis sapiente tempore velit veritatis vero voluptatum.</p>
+        </div>
+    </main>
```

Comment:
Deze main closing tag sluit je af buiten de div waarin je main zit. Code werkt wel alleen is voor de leesbaarheid soms lastig te onderzoeken waar iets begint en eindigt.